grape-active_model_serializers.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rack-test'
   gem.add_development_dependency 'rake'
   gem.add_development_dependency 'guard-rspec'
-  gem.add_development_dependency 'rubocop', '0.28.0'
+  gem.add_development_dependency 'rubocop'
```

Comment:
Good call. I kept it in the .gemspec but locked the version [here](https://github.com/ruby-grape/grape-active_model_serializers/commit/2a4be7238700e240b278fb4542ab304d15053b67). Pushed it straight to master, as it's a small change